### PR TITLE
Update disclaimer

### PR DESCRIPTION
**Pull request by MattF-NSIDC**
## Description

See title!

## Checklist

If an item on this list is done _or not needed_, simply check it with `[x]`.

- [x] Config lockfile updated (`inv config.export > qgreenland/config/cfg-lock.json`)
- [x] Version bumped if needed (`bumpversion (major|minor|patch|prerelease|build`)
- [x] CHANGELOG.md updated
- [x] Documentation updated if needed
- [x] New unit tests if needed

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,17 +20,42 @@ QGreenland Team to data and user communities.
...
+the 'Greenland coastlines 2017' layer as the best approximation reference layer
+for geolocating Greenland's coastline. Note that some data were transformed
+from their native data formats, projections, and resolutions for inclusion
+within QGreenland. The included metadata (>Layer Properties >Metadata >History)
+contains provenance information on any transformations. QGreenland should not
+be used as a sole navigational aid while performing remote research activities
+in Greenland. Always bring appropriate safety and navigational aids (including
+hard-copies of topographic maps) when visiting the field.
+
+QGreenland is a data-viewing platform, and the QGreenland Team does not create
```
Comment (trey-stafford) [inline README.md:44]: This part down looks like a duplicate of information above.

File: README.md [hunk 2]
```diff
@@ -20,17 +20,42 @@ QGreenland Team to data and user communities.
...
 navigational aids (including hard-copies of topographic maps) when visiting the
 field.
 
-QGreenland's layers may contain errors from the original data providers.
-QGreenland makes no guarantees about the accuracy and validity of data
-contained in QGreenland. For example, some layers may not perfectly align with
-each other due to unidentified georeferencing issues. We recommend using the
-'Greenland Coastlines' layer as the reference layer for geolocating Greenland's
-coastline.
+QGreenland is a data-viewing platform, and the QGreenland Team does not create
```
Comment (trey-stafford) [inline README.md:28]: I think we should change this to `QGreenland is a data-viewing and analysis platform`.

Comment (MattF-NSIDC) [reply README.md:28]: 9c34bea9

File: README.md (hunk 1 above)
Comment (MattF-NSIDC) [reply README.md]: 9c34bea9

**Review by trey-stafford — APPROVED** [review]

**Pull request merged**